portfolio/src/main/webapp/myProjects.html

```diff
@@ -0,0 +1,115 @@
+                rent and return movies.</td>
+              <td></td>
+            </tr>
+            <tr>
+              <td>Mar 2019 - Jun 2019</td>
+              <td>Directed Graph Structure with Algorithms</td>
+              <td>Part of the subject Data Structure and Algorithms at UTS (individual project). 
+                Created using C++.</td>
+              <td></td>
+            </tr>
+            <tr>
+              <td>Jul 2019 - Oct 2019</td>
+              <td>E-shop - Computer devices</td>
+              <td>Part of the subject Software Engineering Studio 1A at UTS (group project). 
+                Created using Angular (HTML, CSS, JavaScript). Enable customers to search,
+                browse and buy computer devices.</td>
+              <td></td>
+            </tr>
+            <tr>
+              <td>Mar 2020 - Jun 2020</td>
+              <td>Electrical Forecast Database</td>
+              <td>Part of the subject Database Programming at UTS (individual project). 
+                Created using PL/SQL. Enable to do a daily forecast of around 50,000 
+                engery requirements.</td>
+              <td></td>
+            </tr>
+            <tr>
+              <td>Jul 2020 - Oct 2020</td>
+              <td>E-Library System</td>
+              <td>Part of the subject Software Engineering Studio 1B at UTS (group project). 
+                Created using Ruby on Rails. Enable users to browse, borrow, return and renew e-books</td>
+              <td><a href="https://github.com/helenatran/e-library">GitHub</a></td>
+            </tr>
+            <tr>
+              <td>Jul 2020 - Oct 2020</td>
+              <td>IOU - favour exchange website</td>
+              <td>Part of the subject Advanced Internet Programming at UTS (group project). 
+                Created using MERN (MongoDB, Express, React, Node.js). Enable users to record favours owed among friends</td>
+              <td><a href="https://github.com/helenatran/iou">GitHub</a></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <div class="column side"></div>
+    </div>
+
+    <footer class="page-footer grey darken-3" style="height:65px">
+      <div id="footer-banner" class="center-align">
+        Connect with me: 
+        <span>
+          <a href="https://au.linkedin.com/in/helenaelisabethtran">
+            <img src="https://cdn3.iconfinder.com/data/icons/2018-social-media-black-and-white-logos/1000/2018_social_media_popular_app_logo_linkedin-512.png"/>
+          </a>
+          <a href="https://github.com/helenatran">
+            <img src="https://cdn2.iconfinder.com/data/icons/black-white-social-media/64/github_social_media_logo-512.png"/>
+          </a>
+        </span>
+      </div>
+      <div class="footer-copyright grey darken-3">
+        <div class="container center-align">
+        All photos displayed have been taken by me
+      </div>
+    </footer>
+  </body>
+</html>
```

Comment:
Add a new line at the end of the file. (I found out it's because a lot of old command line tools like grep and cat used to assume there was a newline at the end of the file, and some tools still assume that)